src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -73,8 +73,13 @@ class FacilityTypeDropdown extends Component {
   */
 
   render() {
+    const locationOptions = this.props.showCommunityCares
+      ? this.props.locationOptions
+      : this.props.locationOptions.filter(
```

Comment:
This isn't great but it should only fire when it's not in production. I think driving this with a list of constants is fine but we should probably just remove the indirection and add it to the default state of the component (this component isn't reusable anyway).